### DISCUSSION TITLE
Multiple editors

### DISCUSSION THREAD

**Discussion by lzxlzxlzxl** [ueberdosis/tiptap#3249] [General Questions]
Multiple editors in the same interface, only the last editor can be used，who knows how to solve。

**Comment by vinicius-eidy** [comment]
I have this issue recently, and my solution was pass a property (tiptapName) to give a name to each tiptap, and it can be used as an unique identificator, like:
```
:id="`${tiptapName}-tiptap`"
```
So, every modal that I open in that editor, I know that I'm open that specifically editor item.
